testing/scenario_app/bin/run_android_tests.dart

```diff
@@ -186,8 +187,9 @@ Future<void> _run({
         if (isSkiaGoldClientAvailable) {
           final Future<void> comparison = skiaGoldClient!
               .addImg(fileName, goldenFile, screenshotSize: screenshot.pixelCount)
-              .catchError((dynamic err) {
-            panic(<String>['skia gold comparison failed: $err']);
+              .catchError((Object error) {
+            logWarning('skia gold comparison failed: $error');
```

Comment:
Not sure what was happening when you threw from an error handler block. Seems reasonable to try